lzy/graph-executor-2/src/main/java/ai/lzy/graph/model/TaskOperation.java

```diff
@@ -0,0 +1,21 @@
+package ai.lzy.graph.model;
+
+import java.time.Instant;
+
+public record TaskOperation(
```

Comment:
what is difference with the `Task` class?
Is it a runtime repr of the running task?

Reply:
Yes, this class is used for running tasks, probably I can merge it with TaskState, but I think, that it's more consistent this way.